src/env.h

```diff
@@ -219,6 +219,7 @@ namespace node {
   V(verify_error_string, "verifyError")                                       \
   V(version_string, "version")                                                \
   V(weight_string, "weight")                                                  \
+  V(windows_hide_string, "hide")                                              \
```

Comment:
I was modelling it after the UV constant name, but I see that `hide_string` would be better, done.